### ISSUE TITLE
Telugu Translation Progress

### ISSUE THREAD

**Issue by tesseralis** [reactjs/te.reactjs.org#1]
## Maintainer List

* @ashr81
* @KoushikKumar
* @daminisatya 

## For New Translators

To translate a page:

1. Check that no one else has claimed your page in the checklist and comments below.
2. Comment below with the name of the page you would like to translate. **Please take only one page at a time**.
3. Clone this repo, translate your page, and submit a pull request!

Before contributing, read the glossary and style guide (once they exist) to understand how to translate various technical and React-specific terms.

Please be prompt with your translations! If you find that you can't commit anymore, let the maintainers know so they can assign the page to someone else.

## For Maintainers

When someone volunteers, edit this issue with the username of the volunteer, and with the PR. Ex:

* [ ] Home Page (@tesseralis) #1

When PRs are merged, make sure to mark that page as completed!

## Core Pages

To do before releasing as an "official" translation. **Please translate these pages first.**

* [x] Home Page (@ashr81) #2 (done)
* [ ] Tutorial (@ChaituKNag) (Inprogress)

### Basics

* [ ] Hello World (@mdrahiem)
* [ ] Introducing JSX (@v891) (Inprogress)
* [ ] Rendering Elements
* [ ] Components and Props
* [ ] State and Lifecycle
* [ ] Handling Events
* [ ] Conditional Rendering
* [ ] Lists and Keys
* [ ] Forms
* [ ] Lifting State Up
* [ ] Composition vs Inheritance
* [ ] Thinking in React

### API Reference

* [ ] React
* [ ] React.Component
* [ ] ReactDOM
* [ ] ReactDOMServer
* [ ] DOM Elements
* [ ] SyntheticEvent
* [ ] Test Utilities
* [ ] Shallow Renderer
* [ ] Test Renderer
* [ ] JS Environment Requirements
* [ ] Glossary

### Navigation

These are the navigation links that appear in the sidebar. Possibly wait until
the corresponding sections are translated to do these.

* [ ] tutorial/nav.yml
* [ ] docs/nav.yml

## Next Steps

These are the next most important translations, ordered by importance:

### Installation

* [ ] Getting Started
* [ ] Add React to a Website
* [ ] Create a New React App
* [ ] CDN Links

### Hooks

Everyone is hooked on hooks! People are really excited for these and it would be great for them to be translated.

  * [ ] Introducing Hooks
  * [ ] Hooks at a Glance
  * [ ] Using the State Hook
  * [ ] Using the Effect Hook
  * [ ] Rules of Hooks
  * [ ] Building Your Own Hooks
  * [ ] Hooks API Reference
  * [ ] Hooks FAQ

### Warnings

These are the pages that you get when you click the links in the console (e.g. https://reactjs.org/warnings/dont-call-proptypes.html). People tend to search these, so **please don't translate the error message itself**.
 
* [ ] dont-call-proptypes
* [ ] invalid-aria-prop
* [ ] invalid-hook-call-warning
* [ ] legacy-factories
* [ ] refs-must-have-owner
* [ ] special-props
* [ ] unknown-prop

### Advanced Guides

* [ ] Accessibility
* [ ] Code-Splitting
* [ ] Context
* [ ] Error Boundaries
* [ ] Forwarding Refs
* [ ] Fragments
* [ ] Higher-Order Components
* [ ] Integrating with Other Libraries
* [ ] JSX In Depth
* [ ] Optimizing Performance
* [ ] Portals
* [ ] React Without ES6
* [ ] React Without JSX
* [ ] Reconciliation
* [ ] Refs and the DOM
* [ ] Render Props
* [ ] Static Type Checking
* [ ] Strict Mode
* [ ] Typechecking With PropTypes
* [ ] Uncontrolled Components
* [ ] Web Components

### FAQ

* [ ] APIs and AJAX
* [ ] Babel, JSX, and Build Steps
* [ ] Passing Functions to Components
* [ ] Component State
* [ ] Styling and CSS
* [ ] File Structure
* [ ] Versioning Policy
* [ ] Virtual DOM and Internals

### Contributing

Priority: Low

* [ ] How to Contribute
* [ ] Codebase Overview
* [ ] Implementation Notes
* [ ] Design Principles

### Components

Components in `src/components` that have some text in them.

* [ ] CodeEditor
* [ ] CodeExample
* [ ] ErrorDecoder
* [ ] LayoutFooter
* [ ] LayoutHeader
* [ ] MarkdownPage

## Additional Translations

These are not the primary translation targets.

* [ ] Blog
* [ ] Community  (@sivakumarkomma) #6  (inprogress)

**Comment by tesseralis** [comment]
@ashr81 @KoushikKumar please use this issue to track your progress!

**Comment by tesseralis** [comment]
@ashr81 @KoushikKumar how is it going? Have you started looking at any pages to translate yet? Please let me know if you need any help. :)

**Comment by ashr81** [comment]
@tesseralis i raise a pull request for home pages. but i am not allowed to edit this issue to update my name over here.

**Comment by ashr81** [comment]
@tesseralis i completed conversion for home page. And updated this issue accordingly.

**Comment by daminisatya** [comment]
@tesseralis @ashr81 Guide me on how I can help with this? 😄

**Comment by ashr81** [comment]
@daminisatya great you choose to contribute. Add in your name to maintainter in the file name: `langs/te.json` on this [repository](https://github.com/reactjs/reactjs.org-translation). Follow this documentation [here](https://github.com/reactjs/reactjs.org-translation#adding-a-maintainer).
After you are added into the repo. Check this issue for any guidance. And make sure you take single page to convert everytime. And also look out for open pull request and review them.

**Comment by ChaituKNag** [comment]
Hey @tesseralis, I raised a PR to add myself as a contributor here, could you please approve 🙃

**Comment by ChaituKNag** [comment]
@ashr81 @daminisatya Guys, I am starting the translation for Tutorial page, please let me know if anyone else is doing it.

**Comment by ashr81** [comment]
@ChaituKNag  you can start with tutorial page. Before starting update the issue with your name beside the page name.

**Comment by daminisatya** [comment]
@ChaituKNag Please go ahead and pick up pages from Tutorial! Thank you so much again!

**Comment by ChaituKNag** [comment]
> @ChaituKNag Please go ahead and pick up pages from Tutorial! Thank you so much again!

I see your name already against the Tutorial page (in this issue description), this is a single page (https://reactjs.org/tutorial/tutorial.html), hope it is fine for me to pick up anyway. Please let me know otherwise.

**Comment by ChaituKNag** [comment]
> @ChaituKNag you can start with tutorial page. Before starting update the issue with your name beside the page name.

Do you mean inside this issue description? I am not sure how to edit the issue description.

**Comment by daminisatya** [comment]
Can you please update this PR? https://github.com/reactjs/reactjs.org-translation/pull/106

I already added your name in the issue description. Feel free to open a pr for it and @ mention me.

**Comment by ChaituKNag** [comment]
> Can you please update this PR? [reactjs/reactjs.org-translation#106](https://github.com/reactjs/reactjs.org-translation/pull/106)
> 
> I already added your name in the issue description. Feel free to open a pr for it and @ mention me.

PR comments addressed, please check once.

**Comment by pvsiddhartha** [comment]
Aww! this is awesome.

**Comment by mdrahiem** [comment]
@arash817 @tesseralis @daminisatya do you guys have any pages for me? Finally I am here. :)

**Comment by ashr81** [comment]
@mdrahiem we have many pages to convert. check this issue and update with your name beside the page name you are trying to convert.

**Comment by mdrahiem** [comment]
@ashr81 Thanks for quick response. Shall I start with Basics --> Hello World? I don't see anybody's name attached to that page.

**Comment by ashr81** [comment]
@mdrahiem you can go ahead with that page. 👍

**Comment by sivakumarkomma** [comment]
hello everyone, I want to contribute for this repo , I have done community support page translation , please do review.
I hope no one is working on community part :)

**Comment by ashr81** [comment]
Hi Siva Kumar, I added your name onto the community translation page under the `Additional Translation` section. For the first comment on this issue. let me know, if you are okay with converting all the community pages(pages under community section from navbar[team, courses etc..,]).

**Comment by sivakumarkomma** [comment]
@ashr81 thanks , I will do translate for community section.

**Comment by AmitSharmamad** [comment]
@ashr81 hi ashrith, i am interested in contributing to this translation. Pleas guide me through. 

ధన్యవాదాలు.

**Comment by ashr81** [comment]
@AmitSharmamad  nice to hear that. 

- Add your name to the maintainer's list [here](https://github.com/mdrahiem/reactjs.org-translation/blob/6894840f79364e7401b892dc329b5590d8d1acfe/langs/te.json). Check this pull request for reference [here](https://github.com/reactjs/reactjs.org-translation/pull/123/files).
-  Pick any page from this issue [here](https://github.com/reactjs/te.reactjs.org/issues/1#issue-414294429). And add your name next to the page. (let me know if you are unable to add).
- Fork the repository. And translate the respective page and commit, push onto your fork and raise a pull request to this repository.

If you are new to Github you can find this instruction [here](https://gist.github.com/Chaser324/ce0505fbed06b947d962) helpful in raising your first pull request.

you will find the instructions [here](https://github.com/reactjs/te.reactjs.org/blob/master/README.md) on running the application locally.

**Comment by kaushik1094** [comment]
@ashr81 @daminisatya @mdrahiem Could any one of you add Me (@kaushik1094) and @jsgiant to maintainers list in https://github.com/mdrahiem/reactjs.org-translation/blob/6894840f79364e7401b892dc329b5590d8d1acfe/langs/te.json and create a PR Thank you 👍

**Comment by ashr81** [comment]
Hello @kaushik1094 

- Add your name to the maintainer's list [here](https://github.com/mdrahiem/reactjs.org-translation/blob/6894840f79364e7401b892dc329b5590d8d1acfe/langs/te.json). Check this pull request for reference [here](https://github.com/reactjs/reactjs.org-translation/pull/123/files).
-  Pick any page from this issue [here](https://github.com/reactjs/te.reactjs.org/issues/1#issue-414294429). And add your name next to the page. (let me know if you are unable to add). This helps in knowing others about whoever working on translating a page.
- Fork the repository. And translate the respective page and commit, push onto your fork and raise a pull request to this repository.

If you are new to Github you can find this instruction [here](https://gist.github.com/Chaser324/ce0505fbed06b947d962) helpful in raising your first pull request.

you will find the instructions [here](https://github.com/reactjs/te.reactjs.org/blob/master/README.md) on running the application locally.

**Comment by v891** [comment]
I would like to contribute to Introducing JSX page.

**Comment by ashr81** [comment]
@v891 , Cool. I added your name beside the Introducing JSX page.

**Comment by BhuvaneshReddy** [comment]
Hi, I would like to contribute.

**Comment by gaearon** [comment]
It looks like this translation has stalled, and is now very outdated.
I'm going to archive this repo, but if someone wants to pick up, please start a new effort.
